### ISSUE TITLE
about post #913 (what about haxball in the future?)

### ISSUE THREAD

**Issue by Wenamka** [haxball/haxball-issues#916]
This great game lose a lot of people every day, so bad news, so many leagues closed bcz no one can make some job (i mean to be an admins, etc) for small community.

I can say about russian community, it was great time at 2011-2012, so many players came from social nets like "Vkontakte" or a lot of players from football managers and football websites joined to haxball. It was cool time, we got 5 divisions with  75-85 teams (one team ~ 6-7 ACTIVE players and im not talking about FA or publicmans who playing for fun and dont care about champs and tournament games). But after in 2014 old players with huge haxball experience starting quit from game, ofc we cant play it everyday and everytime for a different reasons (study in universuty, jobs, familly, etc) and thats normally, its like hobby, just a game (if you have good emotiens and love this game - so you will continue to play it). But if someone left game - then someone should join to it, but thats doesnt work with haxball, bcz game isnt popular, not many updates (we dont need physics update but some usefull features - its so important for game community), so: we havent a new players, and now from big russian community we have 7-8 team from really small fastcup without free agents and new teams cant find some teammates for game, + havent admins for championship (that was huge traditional for us)

So basro just make remake with html5 but we are still havent new players, its same haxball faces, same players, whers new players?

So like i said it before - for new players need an own haxball client and move game to steam, probably makes not free, 2-3 dollars for game - thats enough, with game shop probably (change avatar - 1 dollar and etc). I tried makes some reklame and promote haxball in russian websites but i havent a lot moneys for paying strange youtubers for haxball integration, so expensive for me, i said it to community but it was so funny, they laughed about my opinion. And my idea only for my russianspeakers region (eastern europe: russia, ukraine, belarus) but not for europe or NA.

And i think Basro need makes team of game creators which will be makes new client game with similar physics, need add matchmaking with elo system, makes some of virtual pubs, etc. But who cares? Game was so popular and whats now? 1 room for russian big community?

I think that same story for europe. I saw like closed a big national tournament on ESL, that was hige and great cup, that was same like football WC with groups and KO stage, and that closed in 2015. Why? Because haxball not popular game. Champions leagues have same huge problems, not enough communities for it, not enough teams and players.

So i have a question to Basro: why you remade haxball from flash to html? Isnt for something new, i guess cuz flash rip coming soon or something like that.

I really love this game but i cant play some friendly matches because no one is playing on haxball in my country, i want to play some tournaments for my CIS region. But we havent 400-500 players like 5-6 years ago, we have 35-50 but all of them dont play so often. What should i do? Stopped with haxball like all of my friends and like other players? Whats next? In next year i will playing alone in my own room? Btw idea with headless host could some players for europe save part of commutinity and save some motivation (cuz added personal base of stats).

So sad.

**Comment by dixtel** [comment]
I think that haxball is no more than small game to play time to time with your friends in school or others players and have great time, and idea of paying for this game is very bad.

**Comment by Dot5555** [comment]
in Brasil still have some players, but, the number is decreasing, the switch to html 5 is the principal cause

**Comment by sublime801** [comment]
Browser games aren't played anymore as much as like at past. 

An example:  Transformice.

In 2012-2013 game was played by 120-150k players per day.
Now they are running around 10-20k.

In 2012 the game was less featured, like only moving mices and that stuff. Market was so small, couple items. Played by 150k per day.

2019, the game is fully featured now. Like anything you can imagine is now in the game. And they are around 10-20k.

Much features/updates != More players

In past people playing these games because there is no other thing to do. There was no much online games. YouTube was not popular at all. When they wanted to spent some time on computer they were playing Haxball. But now, there is lot of major online games. I mean for now all games are online based, idk why. All companies ignoring the single player modes now. Plus Youtube and Instagram is huge time spending apps. So complaining about Haxball's player rate and asking for a client is not good idea tbh. Since even Fortnite, PUBG, DOTA and a lot of major games losing players each day, there is no saviour here to save Haxball.

**Comment by Jacob-1998** [comment]
Here's whats interesting.

Haxball's general population in public rooms is bigger since HTML released. It's usually 1250+ in EU night time, whereas in FLASH it was about 750/800. So it has improved about 50%.

This is great, and I think it is because HTML is open to a wider audience than FLASH, since most new browsers don't use FLASH. So we have more possible players in HTML which is why the game has had a nice boost.

But here's the confusing part.

Since HTML, despite the larger public room player-base, the competitive community has shrunk MASSIVELY. There used to be a very lively competitive community in FLASH, in the EU alone at least 8 active leagues, now there's 2, FM and HaxLife. The last HCL (Haxball Champions League) was the first ever in 18 editions, to do a 16 team group stage instead of 32.

So Haxball has gotten bigger by 50% casually thanks to simply being available to more computers.

BUT, the percentage of casual players who then go into competitive Haxball, take the game more seriously and become more active has diminished so much. Why is this? Because of the new gameplay.

Most top players have said it and I have their quotes, the new gameplay in HTML (even before extrapolation) is inferior to the FLASH gameplay. Much less room for creativity or tactics. Much more random, much lower skill ceiling.

If the gameplay remained the same, we would have an even bigger competitive community than FLASH in 2017. Unfortunately basro disagrees, without evidence or reasoning (that I heard), except that he feels its better gameplay in HTML, and his opinion overrides the guys who play the game much more seriously than him, and the evidence - the % of player base who takes game seriously in competitive mode is a great indicator of which physics are better.

FLASH = Lower player base, still had MUCH bigger competitive scene.

That says everything you need to know.

**Comment by Jacob-1998** [comment]
And of course if you have a bigger competitive scene, it retains more of the player base and helps grow the game even more. They are supportive of each other (casual player base and competitive scene).

**Comment by genkaihaku** [comment]
I will jump away a bit from the topic, because time and indifferent attitude towards the development of the game by the creator doesn't allow me to believe that this will ever make a mainstream game.

I have one question and request. Why, despite the passage of almost two years after switching to the html5 port, the game still doesn't have a more advanced player for replays? Adding the option to share the record via a link and quick view of the goals is really so damn time consuming and difficult to implement innovation? Mr. Basro, I am asking only for such a small nod towards the players, bc now we have to look for solutions on fan sites which are not a guarantee of stability.

For posts above. Just enjoy the game while it's still possible. Everything has a life cycle, and HaxBall is one of those games that has been left to itself from the beginning. Concept with great potential, but not fully used, you have to finally accept this fact. Regards.

**Comment by genkaihaku** [comment]
I will jump away a bit from the topic, because time and indifferent attitude towards the development of the game by the creator doesn't allow me to believe that this will ever make a mainstream game.

I have one question and request. Why, despite the passage of almost two years after switching to html5 port, the game still doesn't have a more advanced player for replays? Adding the option to share record via a link and quick view of the goals is really so damn time consuming and difficult to implement? Mr. Basro, I am asking only for such a small nod towards the players, bc now we have to look for solutions on fan sites which are not a guarantee of stability.

For posts above. Just enjoy the game while it's still possible. Everything has a life cycle, and HaxBall is one of those games that has been left to itself from the beginning. Concept with great potential, but not fully used, you have to finally accept this fact. Regards.

**Comment by AnddyAnddy** [comment]
As a Transformice former player
The game sucks since a major update around 2013 that just destroyed any kind of difficulty the game had. (the skills update)

**Comment by Jacob-1998** [comment]
Nope. If your talking 2013 I don't know I wasn't around, in 2017 and early 2018 it was 750/800 average numbers, HTML increased the player base when it was released, as expected, because its available for more people's computers/browsers.

**Comment by Wenamka** [comment]
https://i.imgur.com/oVMfqHz.png
https://i.imgur.com/gWENYvE.png
https://i.imgur.com/jSWveiB.png

And thats not even in prime time, screens from 2011-2013 years

**Comment by Jacob-1998** [comment]
Yes in 2013. Thats when Flash was way bigger and more common than HTML.

Just before HTML Haxball was released, Flash Haxball had the lowest ever numbers, lower than they are now. It was just that a higher % of those low numbers took the game seriously, so we still had active leagues.

The real question is why such a low % of the player base takes the game seriously, and plays more actively in leagues, compared to how many % did this in Flash.

If the same % of players took HTML seriously as the amount in Flash did in late 2017/early 2018, we would have more leagues and a larger competitive scene since HTML was released, but the opposite happened.

**Comment by balmeo** [comment]
I think the main problem is the absolute toxic community in Haxball which is not beginner-friendly at all. How should the user numbers rise, if a new player gets immediately banned? Or you get banned if you disagree with the admin's opinion? The incredibly rough and often offensive tone in the rooms does the rest. Who should feel comfortable with something like that?

**Comment by Wazarr94** [comment]
Did any of you think about the fact people just get bored from the game ?

Apart from the extrapolation issues, the HTML update has been very good. Without it, the game would have been dead by now.

**Comment by Jacob-1998** [comment]
All these circumstances are true but they apply just the same to Haxball in 2017 before HTML came out.

Whats changed is the gameplay, MOST top players have expressed a desire for the old gameplay, because this new gameplay is slightly different, but that slight difference has made a massive negative impact on competitive games. The skill ceiling is MUCH LOWER, there's a massive margin for error so skills and creative plays are not rewarded with goals, the extra randomness of spams in HTML compared to Flash means any deep tactical play is not so helpful.

Its just not a serious game anymore, its been dumbed down and top players from Rq* to Hulk to Tortogol to naho to Pringles to Deliric and MANY MORE have expressed this concern about the new gameplay.

Because of this, we have a competitive scene which is 1/4 the size it used to be, despite HTML having larger overall player base than Flash had at the end (late 2017/early 2018), these facts tell you everything you need to know basro. You have your opinion that the Input Delay is worse or hardly effects Haxball gameplay, but please listen to the top players who have 1000's hours of top level experience over you, we know what we are talking about when it comes to gameplay. 

Just like you were wrong about extrapolation not giving advantage (look how all the guys using extrapolation for advantages freaked when you tried removing it, wouldn't happen if extrapolation wasn't such a significant booster), you might also be wrong about this issue.

If you fix the gameplay to MATCH Flash identically, we will have a Flash size competitive community again, and a larger and more active competitive scene can only help the casual player base grow as well.

Higher % player-base enter Competitive Scene -> Competitive Scene becomes bigger -> More players want to come to Competitive Scene, because it looks like a deeper competitive community and more fun -> Competitive Scene becomes bigger -> More free advertisement, YouTube promos, people recommending their friends etc.. -> Larger player-base

And the cycle continues.

**Comment by Wenamka** [comment]
> Did anyone think about the fact people just get bored from the game ?
> 
> Apart from the extrapolation issues, the HTML update has been very good. Without it, the game would have been dead by now.

yes but like i said, if old players stopped play thats mean new players should replace them, like in any game, the main problem - haxball its a close community without promotion and reklame, haxball havent new players, its a big problem

**Comment by genkaihaku** [comment]
Instead of rocking in the clouds, you'd better mobilize Basro to at least fully complete the html5 version, improving the rec player and a few other elements. It's not much. I'm still waiting.
